examples/testbed/ui.rs

```diff
@@ -1,436 +1,533 @@
+                                    min_width: Val::Px(100.),
+                                    min_height: Val::Px(100.),
                                     ..default()
                                 },
-                                Text::new("Bevy logo"),
+                                Interaction::default(),
+                                Outline {
+                                    width: Val::Px(2.),
+                                    offset: Val::Px(2.),
+                                    color: Color::NONE,
+                                },
                             ));
                         });
-                });
-
-            // four bevy icons demonstrating image flipping
-            parent
-                .spawn(Node {
-                    width: Val::Percent(100.0),
-                    height: Val::Percent(100.0),
-                    position_type: PositionType::Absolute,
-                    justify_content: JustifyContent::Center,
-                    align_items: AlignItems::FlexEnd,
-                    column_gap: Val::Px(10.),
-                    padding: UiRect::all(Val::Px(10.)),
-                    ..default()
-                })
-                .insert(Pickable::IGNORE)
-                .with_children(|parent| {
-                    for (flip_x, flip_y) in
-                        [(false, false), (false, true), (true, true), (true, false)]
-                    {
-                        parent.spawn((
-                            ImageNode {
-                                image: asset_server.load("branding/icon.png"),
-                                flip_x,
-                                flip_y,
-                                ..default()
-                            },
-                            Node {
-                                // The height will be chosen automatically to preserve the image's aspect ratio
-                                width: Val::Px(75.),
-                                ..default()
-                            },
-                        ));
-                    }
-                });
-        });
-}
-
-#[cfg(feature = "bevy_ui_debug")]
-// The system that will enable/disable the debug outlines around the nodes
-fn toggle_debug_overlay(
-    input: Res<ButtonInput<KeyCode>>,
-    mut debug_options: ResMut<UiDebugOptions>,
-    mut root_node_query: Query<&mut Visibility, (With<Node>, Without<ChildOf>)>,
-) {
-    info_once!("The debug outlines are enabled, press Space to turn them on/off");
-    if input.just_pressed(KeyCode::Space) {
-        // The toggle method will enable the debug overlay if disabled and disable if enabled
-        debug_options.toggle();
+                }
+            });
     }
+}
 
-    if input.just_pressed(KeyCode::KeyS) {
-        // Toggle debug outlines for nodes with `ViewVisibility` set to false.
-        debug_options.show_hidden = !debug_options.show_hidden;
-    }
+mod slice {
```

Comment:
This slice scene would be more useful if it used an asymetrical border image.

Reply:
Do you know if there's already such a texture in the repo?

Reply:
There's `numbered_slices.png` which is used in the `ui_texture_slice_flip_and_tile` example.